mxcube3/ui/components/SampleQueue/TaskItem.js

```diff
@@ -196,20 +196,21 @@ export default class TaskItem extends Component {
             <form>
               <div className="form-group row">
                 <label className="col-sm-12">File path:</label>
-                <div className="col-sm-12" style={{ display: 'flex' }}>
+                <div className="col-sm-12" style={{ display: 'flex' }} >
```

Comment:
I have not changed form-input and style, I will try to update anyway.
I use the react-clipboard library just for simplicity, I will try to do the same with standard html5